tests/tracking/test_utils.py

```diff
@@ -115,7 +115,9 @@ def test_get_store_sqlalchemy_store(tmp_wkdir, db_type):
     env = {
         _TRACKING_URI_ENV_VAR: uri
     }
-    with mock.patch.dict(os.environ, env), patch_create_engine as mock_create_engine:
+    with mock.patch.dict(os.environ, env), patch_create_engine as mock_create_engine,\
```

Comment:
Mock out initialization logic that attempts to connect to a DB (so that we don't try to connect to non-existent MySQL, postgresql DBs etc)